embedded-postgresql/src/main/java/com/playtika/test/postgresql/PostgreSQLProperties.java

```diff
@@ -40,4 +40,5 @@
     String database = "test_db";
     String startupLogCheckRegex;
     String initScriptPath;
+    boolean usePullAlwaysPolicy = false;
```

Comment:
i believe it can move to common properties, so all containers can enjoy this feature

Reply:
why not, added a new PR https://github.com/Playtika/testcontainers-spring-boot/pull/684